test/surf-extraction.Tests.ps1

```diff
@@ -116,6 +116,23 @@ Error when opening template file 'missing.json': no such file or directory
         $extractorTemplate = (Join-Path $PSScriptRoot "invalid.json")
         New-ExtractorFromTemplate $extractorName $extractorTemplate | Format-MultilineOutput | Should -Be @"
 Error when reading json template '$extractorTemplate': invalid character 'b' looking for beginning of value
+"@
+        $LASTEXITCODE | Should -Be 1
+    }
+
+    It "should print detailed error information" {
+        $output = Invoke-App create extractor test-name waives.date waives.non-existent 2>&1 | Format-MultilineOutput
+
+        $output | Should -Be @"
+Creating extractor 'test-name'... [FAILED]
+Extractor creation failed with the following error: Invalid Extractor Template
+
+Module (not found):
+    The module waives.not-there does not exist.
```

Comment:
Looks like this should be `waives.non-existent`, per [line 124](https://github.com/CloudHub360/ch360.go/pull/117/files#diff-23dc87ee894b7deb7a3ce19f15306085R124) above.